content/topics/Analyze/causal-inference/rdd/rdd-discrete-running-variable.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "RDD with Discrete Running Variables"

```

Comment:
This building block is based on Kolesar/Rothe. It may be good to also refer to Noack/Rothe, available here http://www.christophrothe.net/papers/robust_fuzzy_oct2023.pdf and forthcoming in Econometrica.

Reply:
Hi! I found the paper by Noack and Rothe very interesting, but I see that it focuses on providing bias-aware and honest confidence sets for _fuzzy_ RD designs with discrete assignment variables, whereas I had thought of concentrating on sharp designs in this BB, and it is already quite long. Do you think it would make sense to write an additional (smaller) BB as an extension to this one, focusing on fuzzy designs and based on Noack and Rothe's paper?